tracer/src/Datadog.Trace/TracerManager.cs

```diff
@@ -429,6 +429,9 @@ private static async Task WriteDiagnosticLog(TracerManager instance)
                     writer.WritePropertyName("iast_enabled");
                     writer.WriteValue(Datadog.Trace.Iast.Iast.Instance.Settings.Enabled);
 
+                    writer.WritePropertyName("iast_deduplication_enabled");
+                    writer.WriteValue(Datadog.Trace.Iast.Iast.Instance.Settings.DeduplicationEnabled);
```

Comment:
Should we add telemetry as well (cf TelemetryCollector)?

Reply:
In ConfigurationTelemetryCollector I see that some settings are added, but not all of them. For Iast, we have only added the IAST_ENABLE setting. More detail would be desirable for telemetry?

Reply:
After discussing it with @robertpi, we will not add it to the telemetrycollector. Thanks anyway for the suggestion!